app/js/components/labOrderEntry/DraftOrder/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import DraftList from './DraftList';
+
+export class LabDraftOrder extends React.Component {
+  render() {
+    const { draftLabOrders, disableCancelButton, disableSaveButton } = this.props;
+    return (
+      <div className="draft-spacing draft-lab-layout">
+        <h5 className="h5-draft-header">
+          Unsaved Draft Orders ({draftLabOrders.length})
```

Comment:
I know you're doing `draftLabOrders.Length` twice in this file, so it's not really much of a biggie but a good approach would be to just declare it once and use everywhere so you don't get to do a `.length` every time. sweet still

Reply:
Done